lib/limit.js

```diff
@@ -14,12 +14,14 @@ module.exports = function limitParser(input) {
     // convert strings
     if (typeof input === 'string') {
         if (input === 'unlimited') return unlimited;
+        if (!/^\d+$/.test(input)) throw new RequestError('limit must be an integer or "unlimited"');
+        if (isNaN(input)) throw new RequestError('limit must be an integer or "unlimited"');
```

Comment:
```suggestion
        if (Number.isNaN(input)) throw new RequestError('limit must be an integer or "unlimited"');
```
May be we should use more robust version ```Number.isNaN``` here.